types.ts

```diff
@@ -318,3 +318,17 @@ export type WarningNote = DisclosedNote | UndisclosedNote;
 export interface WarningNotes {
   notes: Array<WarningNote>;
 }
+
+export interface PersonCaseStatus {
+  personId: number;
+  caseStatuses: Array<CaseStatus>;
+}
+
+export interface CaseStatus {
+  id: number;
+  type: string;
+  subType: string;
```

Comment:
can we be more explicit about the possible values here? use enums?

Reply:
I think these are being redefined by the team and should be strings in the future.